podium-common/src/main/java/nl/thehyve/podium/common/service/dto/ExternalRequestRepresentation.java

```diff
@@ -0,0 +1,20 @@
+package nl.thehyve.podium.common.service.dto;
+
+import lombok.Data;
+import org.springframework.cloud.cloudfoundry.com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.Map;
+
+@Data
+public class ExternalRequestRepresentation implements Serializable{
+
+    @JsonProperty("URL")
+    String url;
+    String humanReadable;
+
+    ArrayList<Map<String, String>> collections;
```

Comment:
ArrayList? And I miss documentation here. What are these fields for, what do they mean?